connector_ecommerce/__openerp__.py

```diff
@@ -20,47 +20,14 @@
 ##############################################################################
 
 {'name': 'Connector for E-Commerce',
- 'version': '2.2.0',
+ 'version': '3.0.0',
  'category': 'Connector',
```

Comment:
Why not setting this as hidden category for not being showed by default in the list?